config/viper.go

```diff
@@ -64,7 +64,7 @@ func setSpvWalletDefaults() {
 
 // setHashDefaults sets default values for hash.
 func setHashDefaults() {
-	viper.SetDefault(EnvHashSalt, "bux")
+	viper.SetDefault(EnvHashSalt, "spv-wallet")
```

Comment:
I thought that we will make it more user oriented (I mean custom salt for each user) but actually this change enables us to easily restore old configuration, just by providing hash salt config.

Reply:
Yes, If someone do not want to remove old users HASH_SALT env will do the work